### ISSUE TITLE
Bad tumour and normal count calculations from mutect VCFs

### ISSUE THREAD

**Issue by morungos** [mskcc/vcf2maf#35]
We've seen an issue where the VCF data from Mutect seems not to be handled correctly. Specifically, the `AD` field is used, and even if it provides two values, the logic carries on and attempts various other ways to calculate the depths. In the case of Mutect, the ends up overwriting these two values with a single value calculated from `FA` and `DP`, therefore we have data loss from the `AD` data.

**Comment by ckandoth** [comment]
Thanks for reporting this. I have seen that the AD values from MuTect often add up to >DP... which leads to some workarounds and data loss. I'll revisit this. Do you have some reliable documentation on what exactly is reported in the MuTect VCF?

**Comment by ckandoth** [comment]
@morungos I pushed a fix in the master branch. Tested fine on the included `data/test_mutect.vcf`. Can you test it on your dataset? I'd really appreciate the feedback. If the bug remains, then please provide a sample variant I can use to debug.

**Comment by morungos** [comment]
I'll look more later and get back on this, but for now, I'd say we did the dirty thing and tweaked to assume that everything that's not alt is ref, so we calculated both counts from the frequency, using 1 - frequency for the ref. From what I could see, there's some logic to use this assumption in cBioPortal anyway (which is where we're importing to) so it didn't seem too bad. Thanks for the push and I'll review against our data when I'm back in the office.
